docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,186 @@
+---
+comments: true
+---
+
+# Deploying Applications with Flux and Keptn
+
+Flux is a tool that keeps Kubernetes clusters synchronized with sources
+of configuration (such as Git repositories) and automates updates to
+configuration when new code is deployed.
+
+This section shows how to add Keptn to an existing Flux use case that runs
+[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/)
+to make it simpler and more straight-forward.
+
+## High-level structure of the Git repository
+
+Since Flux uses a GitOps approach for continuous delivery, the Git
+repository structure for our use case looks like the following:
+
+```markdown
+├── apps
+│   └── dev
+│       ├── podtato-head.yaml
+│       └── kustomize.yaml
+└── clusters
+    └── dev
+        ├── flux-system
+        │   ├── gotk-components.yaml
+        │   ├── gotk-sync.yaml
+        │   └── kustomize.yaml
+        ├── podtato-head-source.yaml
+        └── podtato-head-kustomization.yaml
+```
```

Comment:
If I am already running Flux, is this what I have set up or is it different.  I looked at the doc you linked about Flux pre and post deployment jobs and it doesn't look like this -- but perhaps that is my ignorance.

Reply:
That's the point of this guide, with Keptn you do not need to set up pre and post deployment directories in the git repo, because Keptn has a mechanism how to run pre/post-deployment tasks -> with Keptn it's easier to configure

Reply:
Okay, I took a crack at adding this -- see if I got it right.

Reply:
yep, accepted!